evm_loader/test_keccak_perf.py

```diff
@@ -0,0 +1,42 @@
+from solana.transaction import AccountMeta, TransactionInstruction, Transaction
```

Comment:
We will be evaluating the performance gains by running Uniswap tests, so these tests do not need to be implemented.